### DISCUSSION TITLE
Unclear where the plugin gets "Sources" and "Destinations" from...

### DISCUSSION THREAD

**Discussion by tbalbers** [Alef-Burzmali/netbox-data-flows#23] [Q&A] [answered]
If I try to add a new Data Flow, I have to choose "Sources" and "Destinations" in the form. However, there's nothing to choose from. So where do I create the objects that can be chosen in these fields?

I've already added devices and IP addresses, but that doesn't help.

Thanks,

**Comment by tbalbers** [comment, marked as answer]
A user in the slack channel helped me out:
Create an object under object aliases.  From there you can select the address (again already assigned to a virtual machine or device).  Then back in your data flow, you can set the destination to that object alias

**Comment by Alef-Burzmali** [comment]
Hello,
Thanks for pointing out the existence of the slack channel :) I'll try to visit it some times.

I will add ObjectAlias as a prerequisite object of dataflows to make it clearer.

For those who want additional info on the reasons behind the object aliases used as source/destination, cf. issue #24.
